### DISCUSSION TITLE
Parameter problem in InstallAddonsPermission

### DISCUSSION THREAD

**Discussion by Zagrthos** [mozilla/policy-templates#1026] [Q&A] [answered]
I have a question regarding the InstallAddonsPermission policy.

I've set the InstallAddonsPermission to true and the AllowList to "https://addons.mozilla.org" but when I open Firefox and go to about:policies then there's the following error:

_Ignoring parameter "https://addons.mozilla.org" - not a valid origin_

Do you know how I can work around this or do you have other solutions?

**Comment by grueneedv** [comment]
Would like to allow ONLY Extensions from addons.mozilla.org to be installed. 
Tried InstallAddonsPermission with false and allowed the domain https://addons.mozilla.org/ which is shown on the policies tab but when I try to install an extension from mozilla it just says not working in red on the site. From e.g. github.com it tells me "not allowed". Any Idea?

**Reply by Zagrthos** [reply]
Your problem sounds like you've disabled the installation permission completely. Try setting the InstallAddonsPermission to true and try again

**Comment by mkaply** [comment, marked as answer]
If you want to only alllow extensions from addons.mozilla.org, you should use the ExtensionSettings policy only:

```
{
  "policies": {
    "ExtensionSettings": {
      "*": {
        "install_sources": ["https://addons.mozilla.org/*"]
      }
    }
  }
}
```

I don't recommend using the InstallAddonsPermission at all.

**Comment by grueneedv** [comment]
Thanks. This worked like a charm.

**Comment by mkaply** [comment]
@Zagrthos My code above should work for you as well. Are you able to try it?

**Reply by Zagrthos** [reply]
Works! Thank you!
